api/v1alpha1/costmanagement_types.go

```diff
@@ -159,6 +159,15 @@ type CostManagementStatus struct {
 	// UploadWait is a field of CostManagement to represent the time to wait before sending an upload.
 	UploadWait *int64 `json:"upload_wait,omitempty"`
 
+	// Last Upload Status
+	LastUploadStatus string `json:"last_upload_status,omitempty"`
+
+	// Last Upload Time
+	LastUploadTime string `json:"last_upload_time,omitempty"`
```

Comment:
Would be nice to additionally capture when the last upload was successful, so you can get an idea for the gap of time and know that you have sent some data.